### ISSUE TITLE
Rhythm stems in tablature should *not* have slashes on them

### ISSUE THREAD

**Issue by GrayCatMusic** [musescore/MuseScore#13802]
Hello,
I have been studying guitar for more than 30 years. I've never encountered a publication that uses slashes on stems for tablature. I've raised this issue in MS3 but it still persists in MS4. Slashes on stems indicate tremolo. While there is no single standard for indicating half notes, dotted half notes, etc in tablature, there's generally something that echos what the rhythm looks like in standard notation. For example, if it's a half note, there's some sort of oval or round shape around the number to indicate this. I've attached screenshots showing my workflow to workaround this. My graphics certainly aren't perfect, but they're better than having tremolo slashes on the stems. 

Here's the line after I elongated the stems in the tablature:
<img width="1116" alt="Screen Shot 2022-10-17 at 10 58 51 AM" src="https://user-images.githubusercontent.com/89216233/196229856-d8131b6f-821e-48af-bed9-f693fd28645a.png">

After converting the half notes to quarter notes and making the rests invisible, I added the graphic I made to the tablature: 
<img width="1689" alt="Screen Shot 2022-10-17 at 11 12 58 AM" src="https://user-images.githubusercontent.com/89216233/196230043-e09ba2cc-ea8e-4cde-bdcf-f39a8dd59e6d.png">

With the eventual hope to have it look something like this, without the standard notation (this makes it universal to all ukuleles). 
<img width="1428" alt="Screen Shot 2022-10-17 at 11 13 36 AM" src="https://user-images.githubusercontent.com/89216233/196230201-351da897-8adb-474e-a114-ebc2479eddcb.png">

But now with the parenthesis around the notes in the last measure, my graphic would look silly on top of that to indicate a dotted half note. 

I don't know if it's ok to post pictures from copyrighted publications I have but I can provide examples from Hal Leonard and Mel Bay books if you'd like. Virtually any guitar book that Hal Leonard has with tablature will use something like what I've created. Again, I've *never* seen the slashes on the stems like MuseScore implements. The closest thing would be lute tablature, but that's covered in a separate section of MS of course. And you wouldn't have slashes to indicate longer notes. 

Let me know how I can be of assistance. Thank you.

**Comment by abariska** [comment]
@GrayCatMusic I agree that slashes is not a good approach for whole and half notes, but there's a question how it should be implemented. 
We briefly discussed this point a year ago but it definitely needs a new design and implementation. It will fit to guitar release.
FYI @bkunda
